client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CCRDocumentationIT.java

```diff
@@ -395,6 +401,98 @@ public void onFailure(Exception e) {
         assertTrue(latch.await(30L, TimeUnit.SECONDS));
     }
 
+    public void testForgetFollower() throws InterruptedException, IOException {
```

Comment:
I think the hlrc docs itself are missing? Test looks good though.

Reply:
I pushed 63721a459d54adf7517adacb12a13d45bac350e4.